docs/use_database.md

```diff
@@ -5,7 +5,6 @@ MOLGENIS allows you to create multiple databases on one server.
 When you go to the root of your server you will see a listing of the databases. If you don't have permission to see any of
 the databases then a warning will be shown.
 
-If you cannot find it please go to http://servername/apps/central
-
-If you have admin permissions you can create new databases using the '+' sign.
+If you cannot find it please go to [https://SERVERNAME/apps/central](../apps/central/)
```

Comment:
dit is geen valide hyperlink dus zou hier geen bookmark van maken. Je kunt docs namelijk ook bekijken buiten server om, hiero https://molgenis.github.io/molgenis-emx2/#/

Reply:
bookmark verwijdert, uitleg toegevoegd waar de databases te vinden